packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -88,7 +215,7 @@ export default function (program: Command) {
     .asyncActionProjectDir(buildAction, { checkConfig: true });
 
   program
-    .command('build:status')
+    .command('build-status [project-dir]')
```

Comment:
It could be worth adding the following options to filter the list:
- by platform - iOS or Android
- by build id - so we only get a build details output (rather than a list)